lemarche/tenders/constants.py

```diff
@@ -76,3 +76,13 @@
     (SURVEY_NOT_ENCOURAGED_ONLY_BY_US, "Un outil parmi d’autres"),
     (SURVEY_YES, "Oui"),
 )
+
+STATUS_DRAFT = "1"
+STATUS_PUBLISHED = "2"
+STATUS_VALIDATE = "3"
+
+STATUS_LIST = (
```

Comment:
```suggestion
STATUS_CHOICES = (
```
pour homogénéiser avec les autres CHOICES (ce n'est pas une liste à proprement parlé d'ailleurs)